### PR TITLE
Release amd64 and arm64 versions

### PR DESCRIPTION
**Pull request by orblazer**
Hello,
This PR add support for release both `amd` and `arm` 64 version. I have make it modular to easily add more release.
This also remove useless `rel` folder from the repo.

This code was inspired by https://github.com/VictoriaMetrics/VictoriaMetrics.

Note: i havent tried if this broke following step : `Upload image tarballs to GCS`

Thanks

### CODE REVIEW

File: rel/etc/systemd/system/nftables-exporter.service
Comment (majst01) [inline rel/etc/systemd/system/nftables-exporter.service:1]: Why did you remove this ?

**Comment by majst01** [comment]
@vknabel would you please check if this is valid

File: Makefile [hunk 1]
```diff
@@ -6,35 +6,69 @@ VERSION := $(or ${GITHUB_TAG_NAME},$(shell git describe --tags --exact-match 2>
...
+	cp bin/nftables-exporter-$(GOOS)-$(GOARCH) rel/usr/bin/nftables-exporter
 	cp systemd/nftables-exporter.service rel/etc/systemd/system
 	cd rel \
-	&& tar -cvzf nftables-exporter.tgz usr/bin/nftables-exporter etc/systemd/system/nftables-exporter.service \
-	&& mv nftables-exporter.tgz .. \
-	&& cd -
+		&& tar --transform="flags=r;s|-$(GOOS)-$(GOARCH)||" -cvzf nftables-exporter-$(GOOS)-$(GOARCH).tgz \
+			usr/bin/nftables-exporter etc/systemd/system/nftables-exporter.service \
+		&& mv nftables-exporter-$(GOOS)-$(GOARCH).tgz .. \
+		&& cd -
```
Comment (majst01) [inline Makefile:74]: please keep identation

**Comment by majst01** [comment]
Hi, thanks for your contribution, it would be nice if you can align this multi arch build the way we build one of our binaries like seen here: https://github.com/metal-stack/metalctl/blob/master/Makefile

**Comment by orblazer** [comment]
Hello,
I think it's now following your standard.
